internal/kafka/test/common/kafka_pollers.go

```diff
@@ -46,8 +46,13 @@ func WaitForNumberOfKafkaToBeGivenCount(ctx context.Context, db *db.ConnectionFa
 				currentCount = list.Size
 				return currentCount == count, err
 			}
-		}).
-		Build().Poll()
+		})
+
+	if len(customize) > 0 {
+		pb = customize[0](pb)
```

Comment:
why do this only execute the first function? should it iterate and execute all of the customize functions passed?

Reply:
No. I used the `var args` only to make the parameter optional and avoid breaking all the tests.
Only the first one is used.

Reply:
since there's no stopping people from passing multiple functions here, i think it would be great to iterate and apply all but since we don't have a use case for that at the moment, i'm fine with this and we can revisit at a later point if needed 👍